persistence/src/main/java/no/difi/meldingsutveksling/persistence/type/Iso6523Type.java

```diff
@@ -3,16 +3,16 @@
 import no.difi.meldingsutveksling.domain.Iso6523;
 import org.hibernate.dialect.Dialect;
 import org.hibernate.type.AbstractSingleColumnStandardBasicType;
-import org.hibernate.type.DiscriminatorType;
-import org.hibernate.type.descriptor.sql.VarcharTypeDescriptor;
+//todo this does not seem to exist import org.hibernate.type.DiscriminatorType;
```

Comment:
Det ser ut som Hibernate har gått vekk fra denne bruken av discriminator type. kan finne det i docs på version 5.6 under 2.3.6. Custom BasicTypes
https://docs.jboss.org/hibernate/orm/5.6/userguide/html_single/Hibernate_User_Guide.html

mens på 6.6 jeg ser ingen tegn til denne classen
https://docs.jboss.org/hibernate/orm/6.6/introduction/html_single/Hibernate_Introduction.html#compositional-basic-types

under 3.13. Compositional basic types de bruker converters + en @JdbcType tag for å mappe fra databasen og en converter men ingen abstractSingleColumnStandardBasicType lengre

@johannmo

Reply:
@BrettGoreham Ser ut som AbstractSingleColumnStandardBasicType er på veg ut, ja. Me må nok slutta å bruka den.